layouts/partials/posts/series.html

```diff
@@ -0,0 +1,23 @@
+{{ $currentPageUrl := .URL }}
+{{ if .Params.series }}
+<section class="see-also">
+  {{ range .Params.series }}
+    <h3>See also in {{ . }}</h3>
+    {{ $name := . | urlize }}
+    {{ $series := index $.Site.Taxonomies.series $name }}
+    {{ if gt (len $series.Pages) 0 }}
+      <nav class="list">
+        <ul>
+        {{ range first (add $.Site.Params.seriesRelatedCount 1 | default 6) $series.Pages }}
```

Comment:
Just a minor fix here. Making things more clear.
```suggestion
        {{ $maxItems := $.Site.Params.seriesRelatedCount | default 5 }}
        {{ range first (add $maxItems 1) $series.Pages }}
```